code/game/gamemodes/traitor/double_agents.dm

```diff
@@ -1,14 +1,11 @@
 /datum/game_mode/traitor/double_agents
 	name = "double agents"
 	config_tag = "double_agents"
-	restricted_jobs = list("Cyborg", "AI", "Captain", "Head of Personnel", "Chief Medical Officer", "Research Director", "Chief Engineer", "Head of Security") // Human / Minor roles only.

```

Comment:
Can't function how? Traitor AIs just get survive instead of escape, and I'm pretty sure "murder the AI" objectives are possible in traitor.